File: Python/1_Iniciante/1035.py
ipt = input().split()
a = int(ipt[0])
b = int(ipt[1])
c = int(ipt[2])
d = int(ipt[3])

condicoes = [(b>c),(d>a),((c+d)>(a+b)),(c>0),(d>0),((a%2)==0)]

for i in range (len(condicoes)):
    if (condicoes[i]):
        if i == (len(condicoes)-1):
            print('Valores aceitos')
    else:
        print('Valores nao aceitos')
        break

File: Ruby/1_Iniciante/1035.rb
ipt = gets.split()
a = ipt[0].to_i
b = ipt[1].to_i
c = ipt[2].to_i
d = ipt[3].to_i

condicoes = [(b>c),(d>a),((c+d)>(a+b)),(c>0),(d>0),((a%2)==0)]

condicoes.length().times do |i|
    if condicoes[i]
        if i == condicoes.length()-1
            puts "Valores aceitos"
        end
    else
        puts "Valores nao aceitos"
        break
    end
end
